rkt/image/fetchers/common.go

```diff
@@ -0,0 +1,49 @@
+package fetchers
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type InsecureOpts struct {
+	AllowHTTP      bool `json:"allow_http"`
+	SkipTLSCheck   bool `json:"skip_tls_check"`
+	SkipImageCheck bool `json:"skip_image_check"`
+}
+
+type FetcherConfig struct {
+	Scheme        string                         `json:"scheme"`
+	Name          string                         `json:"name"`
+	InsecureOpts  *InsecureOpts                  `json:"insecure"`
+	Debug         bool                           `json:"debug"`
+	Headers       map[string]map[string][]string `json:"headers"`
+	OutputACIPath string                         `json:"output_aci_path"`
```

Comment:
This implies ACI being the image format interface format between rkt and fetcher plugins, don't we want to make the leap and go straight to OCI?